presentation-insights.ipynb

```diff
@@ -2547,6 +2547,58 @@
     "    print(cmd)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Context Managers\n",
+    "\n",
+    "sometimes you need a resource only for a limited time, like opening a file. Context managers allow to automatize this and make sure the resource is closed at the end of the block, no matter what\n",
```

Comment:
"automatize" isn't an English word - "automate" is what you want! (A lot of non-English speakers make that neologism.)

Reply:
Argh, I should have remembered that, I had this discussion before